frontend/src/components/table/classes.ts

```diff
@@ -42,10 +42,10 @@ export const tableClass = (key: string, value?: string) => {
         : tableClasses.unavailable;
     case DashboardColumns.TRAVEL:
       return value === 'yes' ?  tableClasses.travelYes : tableClasses.travelNo;
-    case DashboardColumns.CLASSIFICATION:
-      return value === ClassificationName.EXCLUDED
+    case DashboardColumns.UNION_MEMBERSHIP:
+      return value === Classification.EXCLUDED
         ? tableClasses.excluded
-        : tableClasses.bcgeu;
+        : value === Classification.BCGEU ? tableClasses.bcgeu : tdClass;
```

Comment:
This is a bit strange. What's the requirement here?

Reply:
The color is different depending on the value

Reply:
Sorry, I get that, so it's three different colors? `excluded`, `bcgeu`, and neither?

Reply:
Going to make a change on this to match the new design req's (I spoke with Chevonne earlier and there will now be a color for each value)